app/tests/test_search.py

```diff
@@ -1698,16 +1698,17 @@ def test_search_transferring_body_with_search_term_happy_path_inner_table(
         assert response.status_code == 200
         soup = BeautifulSoup(response.data, "html.parser")
         select = soup.find("select", {"id": "sort"})
-        option = select.find("option", selected=True)
-        option_value = option.get("value")
+        # BUG: value not being saved in dropdown
```

Comment:
is there a followup PR ready for this?

Reply:
@anthonyhashemi not yet but will create a ticket soon and make a PR for the fix

Reply:
I dont think a new ticket is needed, just a PR for that